csv/ParameterDictionary.csv

```diff
@@ -532,7 +532,7 @@ ADCP_PD0,DICT748,ADCP_PD0,adcp_config,"PD7,PD10,PD11,PD12,PD16,PD863,PD611,PD612
 ADCP_PD0,DICT749,ADCP_PD0,adcp_bottom_track_earth,"PD7,PD10,PD11,PD12,PD16,PD863,PD1171,PD1172,PD1173,PD1174,PD1175,PD1176,PD1177,PD1178,PD1179,PD1180,PD1181,PD1182,PD1183,PD1184,PD1185,PD1186,PD1190,PD1191,PD1192,PD1193,PD1194,PD1195,PD1196,PD1197,PD1198,PD1199,PD1200,PD1201,PD1202,PD1203,PD1204,PD1205,PD1207,PD1208,PD1209,PD1210,PD1167,PD1168,PD1169,PD1170,PD1187,PD1188,PD1189,PD1211",PD7,,Science,Bottom Track Data in Earth Coordinates,
 ADCP_PD0,DICT750,ADCP_PD0,adcp_bottom_track_inst,"PD7,PD10,PD11,PD12,PD16,PD863,PD3672,PD3673,PD3674,PD1174,PD1175,PD1176,PD1177,PD1178,PD1179,PD1180,PD1181,PD1182,PD1183,PD1184,PD1185,PD1186,PD1193,PD1194,PD1195,PD1196,PD1197,PD1198,PD1199,PD1200,PD1201,PD1202,PD1203,PD1204,PD1205,PD1207,PD1208,PD1209,PD1210,PD1167,PD1168,PD1169,PD1170,PD1187,PD1188,PD1189,PD1211,PD3675,PD3676,PD3677",PD7,,Science,Bottom Track Data Products,
 ADCP_PD0,DICT751,ADCP_PD0,adcp_bottom_track_config,"PD7,PD10,PD11,PD12,PD16,PD863,PD1160,PD1161,PD1162,PD1163,PD1164,PD1165,PD1166,PD1206",PD7,,Engineering,Bottom Track Configuration Data,
-VADCP,DICT752,VADCP,vadcp_velocity_beam,"PD7,PD10,PD11,PD12,PD16,PD863,PD617,PD621,PD623,PD651,PD664,PD671,PD672,PD673,PD674,PD675,PD676,PD710,PD719,PD720,PD721,PD722,PD724,PD725,PD726,PD727,PD873,PD874,PD875,PD876,PD3811,PD3812,PD3813,PD3814,PD3819,PD3820,PD3827,PD3828,PD2769,PD2770,PD2771,PD2772,PD7983",PD7,,Science,Velocity Data Products,
+VADCP,DICT752,VADCP,vadcp_velocity_beam,"PD7,PD10,PD11,PD12,PD16,PD863,PD617,PD621,PD623,PD651,PD664,PD671,PD672,PD673,PD674,PD675,PD676,PD710,PD719,PD720,PD721,PD722,PD724,PD725,PD726,PD727,PD873,PD874,PD875,PD876,PD3811,PD3812,PD3813,PD3814,PD3819,PD3820,PD3822,PD3827,PD3828,PD2769,PD2770,PD2771,PD2772,PD7983",PD7,,Science,Velocity Data Products,
```

Comment:
Note the only reason this works is that PD3822 maps to a function (PFID31) so this won't affect the cassandra definition.